test/runtests.jl

```diff
@@ -1,24 +1,21 @@
 using Dispatcher
 using ResultTypes
-using Base.Test
+using Compat.Test
 using Memento
 using IterTools
-
+using Compat.Distributed
+using DeferredFutures
 import LightGraphs
 
 using Compat: @__MODULE__
 using ResultTypes: iserror
 
-const LOG_LEVEL = "info"      # could also be "debug", "notice", "warn", etc
-
-Memento.config(LOG_LEVEL; fmt="[{level} | {name}]: {msg}")
 const logger = getlogger(@__MODULE__)
 
 function test_addproc(x::Int; level=LOG_LEVEL)
     ret = addproc(x)
-    @everywhere using Dispatcher
-    @everywhere using Memento
-    @everywhere Memento.config(level; fmt="[{level} | {name}]: {msg}")
+    @everywhere @eval using Dispatcher
+    @everywhere @eval using Memento
```

Comment:
Do we still need this if Memento is being loaded by dispatcher and we aren't changing any settings?

Reply:
We wouldn't, but see my comments above